app/src/test/kotlin/com/intive/picover/images/viewmodel/ImagesViewModelTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.intive.picover.images.viewmodel
+
+import android.net.Uri
+import com.intive.picover.common.coroutines.CoroutineTestExtension
+import com.intive.picover.images.repository.ImagesRepository
+import io.kotest.core.spec.IsolationMode
+import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.matchers.shouldBe
+import io.mockk.Awaits
+import io.mockk.coEvery
+import io.mockk.just
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+
+@OptIn(ExperimentalCoroutinesApi::class)
+internal class ImagesViewModelTest : ShouldSpec({
+	extension(CoroutineTestExtension())
+	isolationMode = IsolationMode.InstancePerTest
```

Comment:
If we would go with onFirstCreate in the viewmodel, we could drop 

`isolationMode = IsolationMode.InstancePerTest`

and just check loading with no mocking for the repo, and then trigger onFirstCreate for the rest 2 tests

Reply:
Conlusion:
We would stick to using `init` instead of `LifecycleViewModel` to start some jobs.